File: Exercícios de Lógica/Java/Extra/Exrcicios_Java/src/Exercicio25.java
/*
Exercicio 025
Faça um Programa que leia três números e mostre o maior e o menor deles.
 */
import java.util.Scanner;
public class Exercicio25 {
    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o primeiro número: ");
        int num1 = scanner.nextInt();

        System.out.print("Digite o segundo número: ");
        int num2 = scanner.nextInt();

        System.out.print("Digite o terceiro número: ");
        int num3 = scanner.nextInt();

        if(num1>=num2 && num1>=num3){
            System.out.println("O maior número é o "+num1);

        }
        else if(num2>=num1 && num2>=num3){
            System.out.println("O maior número é o "+num2);
        }
        else{
            System.out.println("O maior número é o "+num3);
        }
        if(num1<=num2 && num1<=num3){
            System.out.println("O menor número é o "+num1);

        }
        else if(num2<=num1 && num2<=num3){
            System.out.println("O menor número é o "+num2);
        }
        else{
            System.out.println("O menor número é o "+num3);
        }

   scanner.close();
    }
}

File: Exercícios de Lógica/Python/Exercicio25.py
'''
Exercicio 025
Faça um Programa que leia três números e mostre o maior e o menor deles.
'''


num1=float(input("Digite o primeiro número: "))
num2=float(input("Digite o segundo número: "))
num3=float(input("Digite o terceiro número: "))

if num1>num2 and num1>num3:
    print(f"O primeiro número {num1} é o maior")


elif num2>num1 and num2>num3:
    print(f"O segundo número {num2} é o maior")

else:
    print(f"O terceiro número {num3} é o maior")


if num1<num2 and num1<num3:
    print(f"O primeiro número {num1} é o menor")


elif num2<num1 and num2<num3:
    print(f"O segundo número {num2} é o menor")

else:
    print(f"O terceiro número {num3} é o menor")